### PR TITLE
Blogpost to announce that StorSwift joins the LoE

### PR DESCRIPTION
**Pull request by yiannisbot**

### CODE REVIEW

File: docs/blog/2022-03-28-storswift-joining.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: StorSwift joins the League of Entropy
+summary: "StorSwift becomes the newest member of the League of Entropy. We've asked them why they're motivated by drand and decided to contribute - check what they said!"
+date: 2021-09-14
+---
+_"StorSwift becomes the newest member of the League of Entropy. We've asked them why they're motivated by drand and decided to contribute - check what they said!"_
+
+We are excited to announce that [StorSwift](https://storswift.com/), a technology company based in China, which provides cloud computing and storage services, is joining forces with the [League of Entropy](https://leagueofentropy.com/) (LoE)! StorSwift is becoming the 15th member to join the LoE and the first partner joining from China - a significant milestone for the LoE!
+
+We?ve asked StorSwift some key questions that we?ve previously asked other LoE members - see [here](https://drand.love/blog/2021/09/14/the-value-of-drand/) and [here](https://drand.love/blog/2021/10/28/the-value-of-drand-continued/) - on the value of drand. Here are the answers.
```
Comment (mcamou) [inline docs/blog/2022-03-28-storswift-joining.md:10]:
```suggestion
We've asked StorSwift some key questions that we've previously asked other LoE members - see [here](https://drand.love/blog/2021/09/14/the-value-of-drand/) and [here](https://drand.love/blog/2021/10/28/the-value-of-drand-continued/) - on the value of drand. Here are the answers.
```

File: docs/blog/2022-03-28-storswift-joining.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+---
+
+**Why is drand important as a protocol for the next generation of the Internet in your opinion?**
```
Comment (mcamou) [inline docs/blog/2022-03-28-storswift-joining.md:14]:
```suggestion
**In your opinion, why is drand important as a protocol for the next generation of the Internet?**
```

File: docs/blog/2022-03-28-storswift-joining.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+The drand protocol elegantly provides a truly decentralized robust solution to the random numbers quality problem, which indeed is fundamental for secure computer systems.
+
+---
+
+**Why did you decide to join the League of Entropy? What value do you see in drand that is important for your mission (what convinced you to join)?**
```
Comment (mcamou) [inline docs/blog/2022-03-28-storswift-joining.md:20]: IMO this does not really answer the question, but I guess that it's what they answered.

File: docs/blog/2022-03-28-storswift-joining.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+title: StorSwift joins the League of Entropy
```
Comment (johnnymatthews) [inline docs/blog/2022-03-28-storswift-joining.md:2]:
```suggestion
title: "StorSwift joins the League of Entropy"
```

File: docs/blog/2022-03-28-storswift-joining.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+summary: "StorSwift becomes the newest member of the League of Entropy. We've asked them why they're motivated by drand and decided to contribute - check what they said!"
+date: 2021-09-14
+---
+_"StorSwift becomes the newest member of the League of Entropy. We've asked them why they're motivated by drand and decided to contribute - check what they said!"_
```
Comment (johnnymatthews) [inline docs/blog/2022-03-28-storswift-joining.md:6]:
```suggestion
---

_"StorSwift becomes the newest member of the League of Entropy. We've asked them why they're motivated by drand and decided to contribute - check what they said!"_
```

**Review by johnnymatthews — APPROVED** [review]

**Pull request merged**